src/main/kotlin/com/bridgecrew/ui/vulnerabilitiesTree/CheckovFileTreeNode.kt

```diff
@@ -0,0 +1,37 @@
+package com.bridgecrew.ui.vulnerabilitiesTree
+
+import com.intellij.icons.AllIcons
+import icons.CheckovIcons
```

Comment:
perhaps create a module of icons and export from there both the default and the custom ones you use?

Reply:
I don't see the benefit, I actually like the idea of using project based and platform based icons 😄